R/plotAbundance.R

```diff
@@ -180,7 +180,7 @@ setMethod("plotAbundance", signature = c("SummarizedExperiment"),
         order_rank_by <- match.arg(order_rank_by, c("name","abund","revabund"))
         .check_abund_plot_args(one_facet = one_facet,
                             ncol = ncol)
-        if( !is.null(features) ){
+        if( !is.null(features) & !is.null(rank) ){
```

Comment:
Why `features` is disabled? Also use && instead of &

Reply:
The original line didn't have rank set to null and the check had an error on that line, I was just checking whether this would work. Still trying to find a solution as to why the PR runs are failing whereas the tests are running perfectly.

Reply:
It seems that the failure is not related to this PR. It fails before running the tests. I am not sure why.

Reply:
... Now it ran the tests for the latest commit

Have you tested plotAbundance with rank = NULL. It crashes the computer since there are too many things to plot. (The maximum amount of features is probably around 50-100, but there are almost 20,000).

That might be related to this. However, it is still good to allow users to use plotAbundance with rank=NULL since user might already have subsetted the data.

Reply:
The failure might be related to this

Reply:
I set the rank to `rank = taxonomyRanks(x)[1]` if there's more than 500 features and rank is set to null. Let me know what you think about this idea before I  add a warning message for this particular case and I update the documentation.